File: ProjetoJava/src/main/java/source/projetojava/Torcedor.java
package source.projetojava;

/**
 *
 * @author alejb
 */
public class Torcedor extends Pessoa {
  
  public Torcedor() {

    System.out.println("Torcedor criado");
      
  }

  public Torcedor(String nome, String time) {

    super (nome, time);
    System.out.println("Torcedor criado");
    
  }

  @Override
  public void apresentacao () {
      
    System.out.println(this.nome + " é um torcedor do time " + this.time + ".");
    
  }

}

File: ProjetoPython/Torcedor.py
from Pessoa import Pessoa

class Torcedor(Pessoa):

    def __init__(self, nome, time):
        super().__init__(nome, time)
        print("Torcedor Criado")

    def apresentacao(self):
        print(self._nome, "é um torcedor do time", self._time);